Backend/CarRentalApp/CarRentalApp/Services/Token/ITokenService.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System.IdentityModel.Tokens.Jwt;
+using CarRentalApp.Models.Data;
+
+namespace CarRentalApp.Services.Token
+{
+    public interface ITokenService
```

Comment:
Why do you need this interface?

Reply:
This interface is required by DI pattern